native/src/components/RequestLocationPermissionButton.tsx

```diff
@@ -0,0 +1,63 @@
+import LocationIcon from '../assets/IconPlaceholder.svg'
+import SimpleImage from './SimpleImage'
+import styled from 'styled-components/native'
+import { TouchableOpacity, View } from 'react-native'
+import { checkLocationPermission, requestLocationPermission } from '../utils/LocationPermissionManager'
+import React, { ReactElement, useCallback, useEffect } from 'react'
+import { RESULTS } from 'react-native-permissions'
+
+const ICON_SIZE = 45
+
+const Circle = styled(View)`
```

Comment:
We could use the FAB from react-native-elements: https://reactnativeelements.com/docs/fab/